src/IdeaStatiCa.Plugin/Grpc/Reflection/GrpcReflectionServiceFactory.cs

```diff
@@ -23,6 +23,7 @@ public class GrpcReflectionServiceFactory
 
 		class OpCodeContainer
 		{
+			NullLogger ideaLogger = new NullLogger();
```

Comment:
This does not activate the logging, the logger reference needs to be handed over from the caller to enable logging.